diff --git a/kernel/policy/app_profile.c b/kernel/policy/app_profile.c
--- a/kernel/policy/app_profile.c
+++ b/kernel/policy/app_profile.c
@@ -103,8 +103,10 @@
 
     current->seccomp.mode = 0;
     current->seccomp.filter = NULL;
-    atomic_set(&current->seccomp.filter_count, 0);
-    spin_unlock_irq(&current->sighand->siglock);
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 9, 0)
+	atomic_set(&current->seccomp.filter_count, 0);
+#endif
+	spin_unlock_irq(&current->sighand->siglock);
 
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 11, 0)
     // https://github.com/torvalds/linux/commit/bfafe5efa9754ebc991750da0bcca2a6694f3ed3#diff-45eb79a57536d8eccfc1436932f093eb5c0b60d9361c39edb46581ad313e8987R576-R577
