diff --git a/kernel/reboot.c b/kernel/reboot.c
--- a/kernel/reboot.c
+++ b/kernel/reboot.c
@@ -309,6 +309,10 @@
 extern int ksu_handle_sys_reboot(int magic1, int magic2, unsigned int cmd,
 				 void __user *arg);
 #endif
+#if defined(CONFIG_KSU) && defined(CONFIG_KSU_SUSFS)
+extern int ksu_handle_susfs_reboot(int magic1, int magic2, unsigned int cmd,
+				   void __user *arg);
+#endif
 
 SYSCALL_DEFINE4(reboot, int, magic1, int, magic2, unsigned int, cmd,
 		void __user *, arg)
@@ -318,6 +322,10 @@
 	int ret = 0;
 
 #if defined(CONFIG_KSU) && defined(CONFIG_KSU_MANUAL_HOOK)
+#if defined(CONFIG_KSU_SUSFS)
+	if (ksu_handle_susfs_reboot(magic1, magic2, cmd, arg) > 0)
+		return 0;
+#endif
 	ksu_handle_sys_reboot(magic1, magic2, cmd, arg);
 #endif
 
