diff --git a/kernel/Kbuild b/kernel/Kbuild
--- a/kernel/Kbuild
+++ b/kernel/Kbuild
@@ -49,7 +49,6 @@
 
 kernelsu-objs += feature/adb_root.o
 
-kernelsu-objs += feature/selinux_hide.o
 
 kernelsu-objs += feature/uts_spoof.o
 
diff --git a/kernel/feature/selinux_hide.h b/kernel/feature/selinux_hide.h
--- a/kernel/feature/selinux_hide.h
+++ b/kernel/feature/selinux_hide.h
@@ -1,10 +1,20 @@
 #ifndef __KSU_H_SELINUX_HIDE
 #define __KSU_H_SELINUX_HIDE
 
-void ksu_selinux_hide_init();
-void ksu_selinux_hide_exit();
-void ksu_selinux_hide_drop_backup_if_unused();
-void ksu_selinux_hide_handle_second_stage();
-void ksu_selinux_hide_handle_post_fs_data();
+static inline void ksu_selinux_hide_init(void)
+{
+}
+static inline void ksu_selinux_hide_exit(void)
+{
+}
+static inline void ksu_selinux_hide_drop_backup_if_unused(void)
+{
+}
+static inline void ksu_selinux_hide_handle_second_stage(void)
+{
+}
+static inline void ksu_selinux_hide_handle_post_fs_data(void)
+{
+}
 
 #endif
