diff --git a/kernel/Kconfig b/kernel/Kconfig
--- a/kernel/Kconfig
+++ b/kernel/Kconfig
@@ -2,11 +2,11 @@
 
 config KSU
 	tristate "KernelSU function support"
-	depends on KPROBES && EXT4_FS
+	depends on EXT4_FS
 	default y
 	help
 	  Enable kernel-level root privileges on Android System.
-	  Requires CONFIG_KPROBES for kernel hooking support.
+	  Automatic hooking requires CONFIG_KPROBES; manual integration may disable it.
 	  Requires CONFIG_EXT4_FS for `ext4_unregister_sysfs`.
 	  To compile as a module, choose M here: the
 	  module will be called kernelsu.
