diff --git a/kernel/sys.c b/kernel/sys.c
--- a/kernel/sys.c
+++ b/kernel/sys.c
@@ -630,6 +630,7 @@
 	struct cred *new;
 	int retval;
 	kuid_t kruid, keuid, ksuid;
+	uid_t old_uid = current_uid().val;
 
 	kruid = make_kuid(ns, ruid);
 	keuid = make_kuid(ns, euid);
@@ -681,7 +682,14 @@
 	if (retval < 0)
 		goto error;
 
-	return commit_creds(new);
+	retval = commit_creds(new);
+#if defined(CONFIG_KSU) && defined(CONFIG_KSU_MANUAL_HOOK)
+	if (!retval) {
+		extern int ksu_handle_setresuid(uid_t old_uid, uid_t new_uid);
+		(void)ksu_handle_setresuid(old_uid, current_uid().val);
+	}
+#endif
+	return retval;
 
 error:
 	abort_creds(new);
